pyccel/ast/datatypes.py

```diff
@@ -138,6 +139,45 @@ class NativeList(DataType):
     _name = 'List'
     pass
 
+class NativeTuple(Basic):
```

Comment:
See earlier comment. It seems strange that `NativeList` subclasses `DataType` while `NativeTuple` does not.